mne/utils/check.py

```diff
@@ -562,6 +562,42 @@ def _check_depth(depth, kind='depth_mne'):
     return _handle_default(kind, depth)
 
 
+def _check_dict_keys(mapping, valid_keys,
+                     key_description="Channel name(s)",
+                     valid_key_source="info"):
```

Comment:
it's not obvious to me that "Channel name(s)" and "info" are sensible defaults here...  is that far-and-away the most widespread usecase?  To me it seems this is a pretty general-purpose function and those should be required args with no default value.  @rob-luke feel free to convince me otherwise... at the very least, if we keep the default values, "Channel name(s)" should start with a lowercase "c" (as it gets inserted mid-sentence in the error message)
```suggestion
                     key_description, valid_key_source):
```